schema/config-schema.json

```diff
@@ -0,0 +1,659 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/zowe/zlux/zlux.schema.json",
```

Comment:
Manifest.yaml is in a well-known place (root of an extension)
It will be validated against zowe's builtin schema of it, from the well known place of runtimeDirectory/schema/manifest-schema.json
If a manifest brings nothing new to the table over the generic manifest spec, it does not need an additional schema file present.
However, if it has some new properties, such as interesting `configs` definitions, it should define a `schema` property which is the path within the archive to the schema file for validation purpose.

Reply:
As for github, it seems best to put schemas under the home repo for a component so people can find it easily, hence the app-server schema goes into zlux/schema